src/csharp/Grpc.Core.Xamarin/Grpc.Core.Xamarin.csproj

```diff
@@ -0,0 +1,61 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="..\Grpc.Core\Common.csproj.include" />
+
+  <PropertyGroup>
+    <Authors>The gRPC Authors</Authors>
+    <Copyright>Copyright 2015 The gRPC Authors</Copyright>
```

Comment:
Is it okay to keep this as 2015, or should we go straight to 2020?

Reply:
I think it's ok to keep as is (2015 is what the other nugets have), because it doesn't really matter all that much AFAIK.